src/converters/index.ts

```diff
@@ -82,7 +82,8 @@ export function createUserEntity(user: GitLabUser): Entity {
         webLink: user.web_url,
         username: user.username,
         state: user.state,
-        email: user.email,
+        email:
+          user.email !== '' || !user.email ? user.email : user.public_email,
```

Comment:
I don’t know if the condition should be like `email: user.email === “” || !user.email  ? …` I think you are verifying if the emails is empty, or maybe I am not understanding the idea

Reply:
Can we do just this?

```suggestion
          email: user.email || user.public_email,
```

Reply:
At first I tried doing `user.email || user.public_email` as we talked about but tests kept failing since `user.email` was showing up as `""`. 

I think the main fix here is as @jroblesx and @austinkelleher pointed out, the condition was not quite correct.
Should be something like this and separate the function.

```suggestion
          user.email && user.email !== "" ? user.email : user.public_email,
```